plugins/modules/k8s_log.py

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, Fabian von Feilitzsch <@fabianvf>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: k8s_log
+
+short_description: Fetch logs from Kubernetes resources
+
+version_added: "2.8"
```

Comment:
One more request, completely forgot about it in earlier reviews—can you set this to "1.0.0"? Apparently this won't break anything in ansible-doc, and would be more accurate since that will be the collection version this is added in.

Reply:
done